runtime/flangrti/aarch64-Linux/dumpregs.c

```diff
@@ -111,3 +113,18 @@ getRegs(ucontext_t *u)
   mcontext_t *mc = &u->uc_mcontext;
   return (uint64_t *)mc;
 }
+
+#else
+
+void
+dumpregs(void *regs)
+{
+}
+
+void *
+getRegs(void *u)
+{
+  return (void *)0;
+}
```

Comment:
Is it possible to replace these empty functions with void macros? e.g.
```c
#if defined(HAVE_GREGSET_T)
...
#else
#define dumpregs ((void *) 0)
#define getRegs ((void *) 0)
#endif
```

Reply:
I could but then, I'll have to change other places where `dumpregs` is called with ifdefs.

Reply:
since `void *` is not a function.

Reply:
ping on this

Reply:
`dumpregs` and `getRegs` are only called in one file, so it's not a big deal either way. OK.